scripts/core/enqueue.yml

```diff
@@ -1061,7 +1061,6 @@ returns:
     - $X_RESULT_ERROR_INVALID_SIZE:
         - "`patternSize == 0 || size == 0`"
         - "`patternSize > size`"
-        - "`(patternSize & (patternSize - 1)) != 0`"
```

Comment:
I think this change should be reflected in the equivalent command-buffer entry-point too https://github.com/oneapi-src/unified-runtime/blob/daa3ee1a8922030e655cd7a4a71ed7727f7b0bfc/scripts/core/exp-command-buffer.yml#L411